src/main/java/org/epamTask/Drink.java

```diff
@@ -0,0 +1,51 @@
+package org.epamTask;
+
+public class Drink {
+    private String name;
+    private Double price;
+    private Double volume;
+    private String manufacturer;
+
+
+    public Drink(String name, double price, double volume, String manufacturer) {
+        this.name = name;
+        this.price = price;
+        this.volume = volume;
+        this.manufacturer = manufacturer;
+
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public double getPrice() {
+        return price;
+    }
+
+    public void setPrice(double price) {
+        this.price = price;
+    }
+
+    public double getVolume() {
```

Comment:
When you change type of the field you need also change type in getters and setters (the same in another files)